taskcluster/kinds/clean-corpus/kind.yml

```diff
@@ -68,7 +68,10 @@ tasks:
                 USE_OPUSCLEANER: "{use_opuscleaner}"
                 COMPRESSION_CMD: zstdmt
                 ARTIFACT_EXT: zst
-    
+            # there might be intermittent issues with OpusCleaner, specifically FastText model downloading can fail
+            retry-exit-status:
+                - 1
```

Comment:
Are there reasons other than model downloading that this may exit with 1? Preferably, model downloading should fail with a distinct code to avoid unnecessary retries.

Reply:
I see that OpusCleaner exits with 1 on all exceptions. I lost the failed tasks, it's hard to find them now, but I think it was 1 for fasttext

Reply:
I'm not going to say it blocks landing this, but a nice follow-up would be to catch this specific failure (eg: by analyzing output) and returning a distinct code for it.

Reply:
Agreed it would be nice to reproduce. However, I still think we should handle code 1 because it's how OpusCleaner exists on error and we generally want to retry on those errors. It includes many rules and some of them have external dependencies.